CalculatorG.rb

```diff
@@ -0,0 +1,237 @@
+# Project Calculator (gold)
+
+puts "Choose from the list what operation you would like to do:\n "
+puts " add (+)\n sub (-)\n mult (*)\n div(/)\n exponent (^, **)\n mod (%)"
+user_input = gets.chomp
+
+# Function for addition……………………………………………………………………………………………………………………………………………………
```

Comment:
Good use of comments to separate it out. In a more complicated example you could use the comments to describe the overall functionality of the method.